### PR TITLE
For homework

### PR DESCRIPTION
**Pull request by senya2006**
Pls check my homework

### CODE REVIEW

**Comment by senya2006** [comment]
pls check my homework

File: Homework_6/Task_1.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+my_list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+devided_num = int(input("Enter number you want to be devided: "))
+new_list = []
+for number in my_list:
+    if number % devided_num == 0:
+        new_list.append(number)
+
+print(new_list)
```
Comment (MarinaZelenska) [inline Homework_6/Task_1.py:8]: Formatting string :)

File: Homework_6/Task_2.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+my_list = [1, 5, 68, 0]
+# my_list = [1,2,3,4,6,7,8]
+for i in range(1, len(my_list)):
+    if my_list[i] != my_list[i-1] + 1:
+        print("Non-consecutive number in the list: ", my_list[i])
```
Comment (MarinaZelenska) [inline Homework_6/Task_2.py:5]: Formatting string

File: Homework_6/Task_2.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+        break
+else:
+    print("The list contains consecutive numbers!")
```
Comment (MarinaZelenska) [inline Homework_6/Task_2.py:8]: If I create a longer list, the program work in not a good way
image

Comment (MarinaZelenska) [inline Homework_6/Task_2.py:8]: Need to add new line in the end of file

File: Homework_6/Task_3.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+for i in range(1,6):
+    for j in range(1, i+1):
+        print(j, end=' ')
+    print()
```
Comment (MarinaZelenska) [inline Homework_6/Task_3.py:4]: Need to add new line in the end of file.
Use CTRL + ALT + L after coding

File: Homework_6/Task_2.py (hunk 2 above)
Comment (senya2006) [reply Homework_6/Task_2.py:5]: What do you mean by "Formatting string"?

**Pull request merged**